File: Lab1_0/C#/Class.cs
class StringClass
{
    public string text;

    public StringClass(string t)
    {
        text = t;
    }

    public int Length()
    {
        return text.Length;
    }

    public void ShiftRight()
    {
        if (text.Length <= 1)
            return;

        char last = text[text.Length - 1];
        text = last + text.Substring(0, text.Length - 1);
    }
}

File: Lab1_0/C++/Class.cpp
#include "Class.h"

StringClass::StringClass(std::string t)
{
    text = t;
}

int StringClass::length()
{
    return text.length();
}

void StringClass::shiftRight()
{
    if (text.length() <= 1)
        return;

    char last = text[text.length() - 1];
    text = last + text.substr(0, text.length() - 1);
}